### ISSUE TITLE
7.1.2 update: the suppression file stopped working

### ISSUE THREAD

**Issue by dmitry-weirdo** [dependency-check/DependencyCheck#4765]
**Describe the bug**
After updating from `7.1.1` to `7.1.2`, the false-positives excluded with the file started to fail again.

Failure:
```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.1.2:check (default-cli) on project ins-app: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '8.0': 
[ERROR] 
[ERROR] rhino-1.7.7.2.jar: [sonatype-2018-0590] CWE-611: Improper Restriction of XML External Entity Reference ('XXE')(8.8)
[ERROR] spring-web-5.3.22.jar: CVE-2016-1000027(9.8)
[ERROR] 
[ERROR] See the dependency-check report for more details.
```

Suppression file:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!--
    An advice from https://github.com/spring-projects/spring-framework/issues/24434#issuecomment-1132621725
-->

<!--
Documentation: https://jeremylong.github.io/DependencyCheck/general/suppression.html
-->
<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
    <suppress>
        <notes><![CDATA[Ignored since we are not vulnerable]]></notes>
        <packageUrl regex="true">^pkg:maven/org\.springframework/spring.*$</packageUrl>
        <cve>CVE-2016-1000027</cve>
    </suppress>

    <!-- Known FP, see https://github.com/jeremylong/DependencyCheck/issues/4671 -->
    <suppress>
        <notes><![CDATA[Ignored since it is a known false-positive]]></notes>
        <packageUrl regex="true">^pkg:maven/jakarta.*$</packageUrl>
        <cve>CVE-2022-31569</cve>
    </suppress>

    <!-- todo: remove this when swagger/rhino fixes this CWE -->
    <!--
        Ignore this failure that appeared with Sonartype update at 25.05.2022:
        - rhino-1.7.7.2.jar: [sonatype-2018-0590] CWE-611: Improper Restriction of XML External Entity Reference ('XXE')(8.8)
    -->
    <suppress>
        <notes><![CDATA[TODO: uncomment when swagger/rhino fixes it]]></notes>
        <packageUrl regex="true">^pkg:maven/org\.mozilla/rhino.*$</packageUrl>
        <cwe>611</cwe>
    </suppress>
</suppressions>
```

Maven plugin configuration:
```xml
            <plugin>
                <groupId>org.owasp</groupId>
                <artifactId>dependency-check-maven</artifactId>
                <version>${dependency-check-maven.version}</version>
                <configuration>
                    <suppressionFile>${project.build.outputDirectory}/dependency-check-suppress.xml</suppressionFile>
                    <failBuildOnCVSS>8</failBuildOnCVSS>
                    <formats>
                        <format>junit</format>
                    </formats>
                    <prettyPrint>true</prettyPrint>
                    <nodeAuditSkipDevDependencies>true</nodeAuditSkipDevDependencies>
                    <nodePackageSkipDevDependencies>true</nodePackageSkipDevDependencies>
                    <assemblyAnalyzerEnabled>false</assemblyAnalyzerEnabled>
                    <yarnAuditAnalyzerEnabled>false</yarnAuditAnalyzerEnabled>

                    <!--
                        Links to servers.server.id in Maven's settings.xml.
                        See https://github.com/jeremylong/DependencyCheck/issues/4535#issuecomment-1137096786
                    -->
                    <ossIndexServerId>owasp-oss-index</ossIndexServerId>

                    <!-- Fix of https://github.com/jeremylong/DependencyCheck/issues/4695 -->
                    <retireJsUrl>
                        https://raw.githubusercontent.com/RetireJS/retire.js/33b4076ce87f3898b81af4fc1770a7b65aa54bcb/repository/jsrepository.json
                    </retireJsUrl>
                </configuration>
            </plugin>
```

**Version of dependency-check used**
7.1.2

**Expected behavior**
The suppression XML file is working as in `7.1.1`

**Comment by strangelookingnerd** [comment]
I found that this only applies for me if I run the check on the multi-module where no suppression file is configured.
It works just fine if I run the check on a module itself that has a suppression file defined.

**Comment by jeremylong** [comment]
Thanks for the report. I'll work on the fix shortly.

**Comment by dmitry-weirdo** [comment]
Seems to be fixed in `7.2.0`, thank you.
